static/app/components/events/aiAutofix/types.ts

```diff
@@ -1,17 +1,31 @@
 import type {EventMetadata, Group} from 'sentry/types';
 
 export type AutofixData = {
-  completedAt: string | null;
-  createdAt: string;
-  status: 'PROCESSING' | 'COMPLETED' | 'ERROR';
-  errorMessage?: string;
+  created_at: string;
+  status: 'PROCESSING' | 'COMPLETED' | 'NOFIX' | 'ERROR';
+
+  codebase_indexing?: {
+    status: 'COMPLETED';
+  };
+  completed_at?: string | null;
+  error_message?: string;
   fix?: {
     description: string;
-    prNumber: number;
-    prUrl: string;
-    repoName: string;
+    pr_number: number;
+    pr_url: string;
+    repo_name: string;
```

Comment:
Why can't these be camelCase? Does the backend not use the `CamelCaseSerializer`?

Reply:
this came straight from seer, didn't know that was a thing, will see if I can pass it through

Reply:
Ahh. It would be nice to translate on the backend. Maybe ping @wedamija in #discuss-backend